HeroVault/HeroVault/Features/UserProfile/UserProfileViewController.swift

```diff
@@ -9,7 +9,29 @@ import UIKit
 
 class UserProfileViewController: UIViewController {
     
+    // MARK: IBOutlets
+    
+    @IBOutlet weak private var profileLabel: UILabel!
+    @IBOutlet weak private var userImage: UIImageView!
+    @IBOutlet private weak var userNameLabel: UILabel!
+    @IBOutlet private weak var favouriteCharactersLabel: UILabel!
+    @IBOutlet private weak var favouriteCharactersStarImage: UIImageView!
+    @IBOutlet private weak var favouriteComicsLabel: UILabel!
+    @IBOutlet private weak var favouriteComicsStarImage: UIImageView!
+
+// MARK: IBActions
+    
+    @IBAction func favouriteCharactersButton(sender: Any) {
+        performSegue(withIdentifier: Constants.SegueIdentifierNames.userFavouritesSegueName, sender: sender)
+    }
+    @IBAction func favouriteComicsButton(_ sender: Any) {
```

Comment:
A better way to name this actions is something like "favouriteComicsButtonTapped". Please apply to other IB Actions